### PR TITLE
[WIP] Add origin basic e2e router test to aws installation

### PR DESCRIPTION
**Pull request by imcsk8**
In order to run e2e router tests in aws for the Ingress Operator this basic tests have to be enabled.

### CODE REVIEW

**Comment by imcsk8** [comment]
PTAL @smarterclayton @openshift/sig-network-edge

File: ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml [hunk 1]
```diff
@@ -38,3 +38,7 @@ tests:
     defaultMode set' run-tests
   openshift_installer:
     cluster_profile: aws
+- as: e2e-aws-router
```
Comment (ironcladlou) [inline ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml]: I think we should just modify the existing `aws-e2e` job in here?

Comment (imcsk8) [reply ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml]: I didn't want to touch that one. I'll change it

File: ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ tests:
   container:
     from: src
 - as: e2e-aws
-  commands: TEST_FOCUS='Secrets should be consumable from pods in volume with
+  commands: TEST_FOCUS='should respond with 503 to unrecognized hosts' run-tests
     defaultMode set' run-tests
```
Comment (pravisankar) [inline ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml]: You need to remove this line

**Comment by Miciah** [comment]
Superseded by https://github.com/openshift/release/pull/2100.

**Pull request closed without merging**